### PR TITLE
add new article <Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models>

### PR DESCRIPTION
**Pull request by AlfaRomeo9527**
add new article <Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models>

### CODE REVIEW

**Comment by AlfaRomeo9527** [comment]
Preview link：
https://dev-ui.57blocks.io/blog/preview?url=https://raw.githubusercontent.com/57blocks/website-article/alpha-dev/articles/Image%20Search%3A%20Leveraging%20Image%20Models%2C%20Large%20Language%20Models%2C%20and%20Multimodal%20Models/README.md

File: articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+Where:  
+GTP:denotes the total number of ground truth positives, that is, the number of truth labels/positive samples;  
+n:denotes the number of images involved in the retrieval;  
+rel@k: is a schematic function that is set to 1 when the first retrieval candidate is a similar sample and 0 otherwise.  
+## 3.2 OCR
+As OCR stands for optical character recognition, OCR technology deals with the problem of recognizing all kinds of different characters. Both handwritten and printed characters can be recognized and converted into a machine-readable, digital data format.  
+Think of any kind of serial number or code consisting of numbers and letters that you need digitized. By using OCR you can transform these codes into a digital output. The technology makes use of many different techniques. Put simply, the image taken is processed, the characters extracted, and are then recognized.  
+What OCR does not do is consider the actual nature of the object that you want to scan. It simply.“takes a look” at the characters that you aim to transform into a digital format. For example, if you scan a word it will learn and recognize the letters, but not the meaning of the word.
```
Comment (wholehope) [inline articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md:73]: extra . after simply

File: articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+What OCR does not do is consider the actual nature of the object that you want to scan. It simply.“takes a look” at the characters that you aim to transform into a digital format. For example, if you scan a word it will learn and recognize the letters, but not the meaning of the word.  
+OCR (Optical Character Recognition) typically consists of two crucial steps: Text Detection and Text Recognition.  
+![figure 6 OCR process.png](figure%206%20OCR%20process.png)  
+Fig 6. OCR process  
+### 3.2.1 Text Detection  
+Text detection is the process of identifying and locating the textual regions within an input image or document.This step involves applying specialized models, such as DBNet, CTPN, and EAST, to efficiently and accurately detect the spatial positions of the text.The output of the text detection step is a set of bounding boxes or text region proposals that encapsulate the textual content.  
+**DBNet**   
+DBNet<sup>[7]</sup>employs a differentiable binarization method, which can train the entire network in an end-to-end manner, avoiding the additional post-processing steps required by the traditional proposal-based methods.
```
Comment (wholehope) [inline articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md:82]: need a space after </sup>. There are a few other similar places that need a space.

File: articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+## 4. Conclusion  
+During our image retrieval endeavor, we have diligently addressed the distinctive traits of images pertinent to the business sector, taking into account crucial image features, textual information, and details from both sides of the image. By analyzing and valorizing this information, we have implemented strategies such as reverse image search, textual retrieval, and image search via text. Our deployment of advanced deep learning algorithms encompasses an array of models—ranging from those specialized in computer vision and natural language processing to multimodal algorithms, large language models, and comprehensive multimodal language-vision models. Through systematic enhancement of the performance across each model phase, we have successfully realized a retrieval process that is both effective and precise for images and accompanying text.  
+## 5. Reference
```
Comment (wholehope) [inline articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md:152]: You do not need ##5. Reference usually is not numbered.

Comment (AlfaRomeo9527) [reply articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md:152]: Already modified

**Pull request merged**